pkg/controller/operators/olm/config.go

```diff
@@ -112,6 +113,12 @@ func WithLogger(logger *logrus.Logger) OperatorOption {
 	}
 }
 
+func WithCopiedCSVs(enabled bool) OperatorOption {
```

Comment:
suggestion: wdyt if we change it to just:

```go
func WithCopiedCSVs() {...}
```

meaning it is disabled by default, but you can enable it with this option?

Reply:
Alternatively, you could call it `WithCopiedCSVsEnabled(enabled bool)` or some such, tying in with the previous comment on naming

Reply:
OR, also consider 

```go
func WithoutCopiedCSVs() {}
```

Depending on your defaults...

Reply:
I like the look of the signature, but it wouldn't play as nice with [this code](https://github.com/operator-framework/operator-lifecycle-manager/blob/eabd986fefe848d15c877988a7c4dea5924ae8eb/cmd/olm/main.go#L155-L165) a bit less readable. We would need to create the list of config options before the NewOperator call which isn't as clean IMO. Thoughts?

Reply:
Yeah, that's fair enough.